plugins/ImageGraph/StaticGraph/GridGraph.php

```diff
@@ -22,18 +22,18 @@ abstract class GridGraph extends StaticGraph
 
     const DEFAULT_TICK_ALPHA = 20;
     const DEFAULT_SERIE_WEIGHT = 0.5;
-    const LEFT_GRID_MARGIN = 4;
+    const LEFT_GRID_MARGIN = 20;
     const BOTTOM_GRID_MARGIN = 10;
-    const TOP_GRID_MARGIN_HORIZONTAL_GRAPH = 1;
-    const RIGHT_GRID_MARGIN_HORIZONTAL_GRAPH = 4;
+    const TOP_GRID_MARGIN_HORIZONTAL_GRAPH = 10;
+    const RIGHT_GRID_MARGIN_HORIZONTAL_GRAPH = 20;
```

Comment:
Looking at the changed screenshots it looks like the adjustments were actually only needed for the evolution chart.
The other charts now have additional unused space. Wouldn't it been have possible to only adjust the evolution one?

Reply:
The other charts are also drawn very close to the edge of the image, for example the horizontal bar chart would likely clip the bar totals for numbers with four digits. I think adding few extra pixels of padding around them reduces the likelihood of other clipping issues and will look better if the images are embedded with other text.

Reply:
I guess we would actually need to automatically increase those margins if higher values are meant to be printed.
It might make sense to adjust the UI tests so they use higher values. I think we could achieve that without tracking additional numbers but simply modifying the results. I'll try something an maybe push some changes here to achieve that if easily possible.